packages/link/LinkStyles.module.css

```diff
@@ -0,0 +1,14 @@
+.link {
+  text-decoration: none;
+  color: var(--lido-color-primary);
+
+  :hover {
+    color: var(--lido-color-primaryHover);
+  }
+}
+
+.link.fadedVisited {
+  &:visited {
+    color: var(--lido-color-primaryVisited);
+  }
+}
```

Comment:
```suggestion
  &.fadedVisited:visited {
    color: var(--lido-color-primaryVisited);
  }
}
```

Reply:
I agree that this is a better way to write, but the sample at the top is a style, which I use in every other component, feels like it doesn't worth it to break the pattern here. I guess we will need to figure out a better way for all other components, since the current migration is too style-component oriented, so I would postpone this change.